components/nexus-core/src/main/java/org/sonatype/nexus/internal/security/model/CUserRoleMappingEntityAdapter.java

```diff
@@ -50,7 +50,7 @@
 {
   public static final String DB_CLASS = new OClassNameBuilder()
       .prefix("security")
-      .type("user-role-mapping")
+      .type("user_role_mapping")
```

Comment:
correct, part of OrientDB 2.1.x is a more strict SQL parser: https://groups.google.com/forum/#!topic/orient-database/a8TVNVmas4U